lib/app_home.dart

```diff
@@ -0,0 +1,155 @@
+/*
+ * Copyright (C) 2022 Canonical Ltd
+ *
+ * This program is free software: you can redistribute it and/or modify
+ * it under the terms of the GNU General Public License version 3 as
+ * published by the Free Software Foundation.
+ *
+ * This program is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+ * GNU General Public License for more details.
+ *
+ * You should have received a copy of the GNU General Public License
+ * along with this program.  If not, see <http://www.gnu.org/licenses/>.
+ *
+ */
+
+import 'package:flutter/material.dart';
+import 'package:yaru_icons/widgets/yaru_icons.dart';
```

Comment:
does `yaru_icons/yaru_icons.dart` work if you upgrade yaru_icons to the latest? i recall it was fixed. can be also a separate step because it's not really related to this :)

Reply:
Yes, that works. I'll do in a separate PR though.